docs/Overview.md

```diff
@@ -1,49 +1,54 @@
+* **Efficiency**: We are intensively working on more efficient model tuning from both system level and algorithm level. For example, leveraging early feedback to speedup tuning procedure.
 
->Step 2: [Update model codes](howto_1_WriteTrial.md)
+## Key Concepts
 
->Step 3: [Define Experiment](ExperimentConfig.md)
+* *Experiment*: An experiment is one task of, for example, finding out the best hyperparameters of a model, finding out the best neural network architecture. It consists of trials and AutoML algorithms.
 
+* *Search Space*: It means the feasible region for tuning the model. For example, the value range of each hyperparameters.
 
-<p align="center">
-<img src="./img/3_steps.jpg" alt="drawing"/>
-</p>
+* *Configuration*: A configuration is an instance from the search space, that is, each hyperparameter has a specific value.
 
-After user submits the experiment through a command line tool [nnictl](../tools/README.md), a demon process (NNI manager) take care of search process. NNI manager continuously get search settings that generated by tuning algorithms, then NNI manager asks the training service component to dispatch and run trial jobs in a targeted training environment (e.g. local machine, remote servers and cloud). The results of trials jobs such as model accurate will send back to tuning algorithms for generating more meaningful search settings. NNI manager stops the search process after it find the best models.
+* *Trial*: Trial is an individual attempt at applying a new configuration, for example, a set of hyperparameter values on a model, or a specific nerual architecture.
 
-## Architecture Overview
-<p align="center">
-<img src="./img/nni_arch_overview.png" alt="drawing"/>
-</p>
+* *Tuner*: Tuner is an AutoML algorithm, which generates a new configuration for the next try. A new trial runs with this configuration.
 
-User can use the nnictl and/or a visualized Web UI nniboard to monitor and debug a given experiment.
+* *Assessor*: Assessor analyzes trial's intermediate results (e.g., periodically evaluated accuracy on test dataset) to tell whether this trial can be early stopped or not.
 
-NNI provides a set of examples in the package to get you familiar with the above process. In the following example [/examples/trials/mnist], we had already set up the configuration and updated the training codes for you. You can directly run the following command to start an experiment. 
+Basically, an experiment runs as follows: Tuner receives search space and generates configurations. These configurations will be submitted to training platforms, such as local machine, remote machines, or training clusters. Their performances are reported back to Tuner. Then, new configurations are generated and submitted.
```

Comment:
Consider adding a training platform explanation to the key concepts instead of the “such as” example?

Reply:
Quick question about "SearchSpace", do we want to use them as one term or 2 words?

Agree with PurityFan's on Training Platform's definition. Also, I was thinking Configuration also includes the yalm file's definition for training platform.

Reply:
I can understand what you mean. Need to distinguish between config.yml and configuration.